example/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-    Copyright (c) 2016, 2018 Contributors to the Eclipse Foundation
+    Copyright (c) 2020 Contributors to the Eclipse Foundation
```

Comment:
Since the original file in this directory was moved to [`example/hono-client-examples/pom.xml`](https://github.com/eclipse/hono/pull/1724/files#diff-926b5f64866f1096050ab4477635ec36) one could argue that this is a new file.